components/icon/index.en-US.md

```diff
@@ -0,0 +1,125 @@
+---
+category: Components
+chinese: 图标
+type: Basic
+english: Icon
+toc: false
+---
+
+Meaningful vector graphics, each one can defeat an enemy.
```

Comment:
Its chinese is abstract too. @benjycui , can you help us, what do you want do express?